javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,51 @@
+// Future age calculator
+const yearOfBirth = 1983; //  data type number
+let yearFuture = 2070;
+let age = yearFuture - yearOfBirth;
+console.log(`you will be ${age} years old in ${yearFuture}`);
+
+
+// Dog age Calculator
+
+let dogYearOfBirth = 2004;
+let dogYearFuture =  2024;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears = true)
```

Comment:
This should probably have been 
```js
if (shouldShowResultInDogYears === true)
```
Nice the triple equals, rather than single. Rather than just comparing the value with `true`, javascript will set `shouldShowResultInDogYears ` to `true`, and then check if `shouldShowResultInDogYears` is true. 

Thus the value will always be logged in dog years, even if `shouldShowResultInDogYears` was set to false on line 13.

Reply:
comments are solved!!